File: calculator.py
def add(a, b):
    return a + b

def subtract(a, b):
    return a - b

def multiply(a, b):
    return a * b

def divide(a, b):
    if b == 0:
        print("Error: Cannot divide by zero.")
        return None
    return a / b

if __name__ == "__main__":
    print("Simple Calculator")

    num1 = input("Enter first number: ")
    num2 = input("Enter second number: ")
    operation = input("Enter operation (+, -, *, /): ")

    num1 = float(num1)
    num2 = float(num2)

    if operation == "+":
        result = add(num1, num2)
    elif operation == "-":
        result = subtract(num1, num2)
    elif operation == "*":
        result = multiply(num1, num2)
    elif operation == "/":
        result = divide(num1, num2)
    else:
        print("Invalid operation.")
        result = None

    if result is not None:
        print("Result:", result)

File: calculator.js
function add(a, b) {
    return a + b;
}

function subtract(a, b) {
    return a - b;
}

function multiply(a, b) {
    return a * b;
}

function divide(a, b) {
    if (b === 0) {
        console.log("Error: Cannot divide by zero.");
        return null;
    }
    return a / b;
}

console.log("Simple Calculator");

const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("Enter first number: ", (num1) => {
    readline.question("Enter second number: ", (num2) => {
        readline.question("Enter operation (+, -, *, /): ", (operation) => {
            num1 = parseFloat(num1);
            num2 = parseFloat(num2);

            let result;

            if (operation === "+") {
                result = add(num1, num2);
            } else if (operation === "-") {
                result = subtract(num1, num2);
            } else if (operation === "*") {
                result = multiply(num1, num2);
            } else if (operation === "/") {
                result = divide(num1, num2);
            } else {
                console.log("Invalid operation.");
                result = null;
            }

            if (result !== null) {
                console.log("Result: " + result);
            }

            readline.close();
        });
    });
});